### DISCUSSION TITLE
Installation process

### DISCUSSION THREAD

**Discussion by Co1dSUN** [duhow/wibox#12] [Q&A] [answered]
Hello. It is not clear, where I should copy scripts and other content from folder include before run script update_firmware.sh. Looks like one step after copy update.img to /tmp and run update_firmware.sh is missed. Can you please help me and explain that?

**Comment by duhow** [comment, marked as answer]
I'll guess you mean [building the image](https://github.com/duhow/wibox/blob/main/INSTALL.md#build)?
Also, `scripts` folder is only used for building process, it must not be placed into the Wibox.
If you want to `include` additional content to your custom image (other built binaries), place them in the `include` folder.

**Reply by Co1dSUN** [reply]
Nope. I have build an image and upload it to the box to /tmp/update.img, but I can't get where I should run an update_firmware.sh? Should I upload it to the box also? If so where I should upload it?

**Reply by duhow** [reply]
[update_firmware.sh](https://github.com/duhow/wibox/blob/main/include/bin/update_firmware.sh) is a bundled script, you can copy it manually, or run the commands to flash - I've just updated the documentation with those.

**Reply by Co1dSUN** [reply]
I think I realized where was my mistake. I have to put update.img to folder /tmp on my linux laptop, not on the box.

**Reply by duhow** [reply]
Not at all. You want to "update" Wibox, not your laptop 😂

**Reply by Co1dSUN** [reply]
Well, I still can't get it. There is a folder /wibox-main/include/bin where are several script, include update_firmware.sh. Should I copy it to the box and run there? What about all other scripts from this folder, run.sh, web and so on?
